Core/include/Acts/TrackFinding/FasTrackConnector.hpp

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Is this in `TrackFinding` (rather than `Seeding`) on purpose?

Reply:
I originally put it in there when have python symbol issues, this class is not templated so I can include it as hpp and cpp files and include it in the cmake file. There isnt the same structure in the seeding directory